rad/prelude/error-messages.rad

```diff
@@ -82,11 +82,11 @@
 
 (def uppercase-item
   (fn [item]
-    (lookup item {:issue "Issue" :patch  "Patch"})))
+    (lookup item {:issue "Issue" :patch "Patch" :project "Project"})))
```

Comment:
```
(def uppercase-item [item]
  (titlecase (lowercase-item item))
```

You'll need to add some sort of pattern matching or similar functionality to implement `titlecase`

Reply:
(See above) The flag and item name does not match in this way anymore.